src/components/UploadAttachmentModal.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import AttachmentModal from './AttachmentModal';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const UploadAttachmentModal = props => (
```

Comment:
Lets remove this component and directly use `AttachmentModal` and set the `headerTitle` & `allowDownload` accordingly.

These components are not adding much value.